packages/calypso-build/webpack.config.js

```diff
@@ -53,7 +53,10 @@ function getWebpackConfig(
 	}
 ) {
 	const workerCount = 1;
-	const cssFilename = '[name].css';
+	const cssFilename = path.parse( outputFilename ).name + '.css';
+	const cssChunkFilename = outputChunkFilename
+		? path.parse( outputChunkFilename ).name + '.css'
```

Comment:
These are "template" names. I don't think we want to use path parsing here. I believe they'll tend to look more like URLs than file system paths.

I'd propose an implementation like what [`SourceMapDevToolPlugin` does](https://github.com/webpack/webpack/blob/de81dd096c4f9d2adb03c9a89207a8a7ad499a6f/lib/SourceMapDevToolPlugin.js#L252-L257).

- Pull off a `?query` if found..
- Change the extension to `css`.
- Append the `?query`.

Changing the extension can likely [expect to find `/\.js$/i`](https://webpack.js.org/configuration/output#outputfilename).

Reply:
Done in f62face1da3872babee1027e4d4c77f09b7c5e3b

Reply:
Ah, good point, didn't think of query strings here. (`path.parse().name` was just so temptingly elegant :grimacing: )